src/_P039_Thermosensors.ino

```diff
@@ -938,13 +956,30 @@ float readMax31855(struct EventStruct *event)
                   log += F(" Short-circuit to Vcc");
                 }
               }
-              addLog(LOG_LEVEL_DEBUG, log);
+              addLog(LOG_LEVEL_DEBUG_MORE, log);
             }
           } 
 
-        # endif // ifndef BUILD_NO_DEBUG
+
       }
 
+      # ifndef BUILD_NO_DEBUG
+
+        if (loglevelActiveFor(LOG_LEVEL_DEBUG)) 
+        {
+          String log;
+          if((log.reserve(120u))) { // reserve value derived from example log file
+            log = F("P039 : MAX31855 : ");    
+            log += F("rawvalue, HEX: 0x");
+            log += String(rawvalue, HEX);
+            log += F(" P039_data->sensorFault, HEX: 0x");
+            log += String(P039_data->sensorFault, HEX);
```

Comment:
Just remembered, there is a `formatToHex` function, in StringConverter.

Reply:
You're welcome :)

Reply:
...and used in latest push d908816 to enhance the logfile information.